### ISSUE TITLE
Multiple channels publish

### ISSUE THREAD

**Issue by aleksraiden** [centrifugal/centrifugo#41]
For example: 
```
command = {
        "method": "publish",
        "params": {"channels": ['channelA', 'channelB', 'channelC'], "data": {"input": "xxx"}}
}
```

One message published to many channels at once. For many scenario this dramaticly save traffic and performance (and at client-side too)

E.g. I have rss news server with 100 feed. Any user can subscribe to any feed or array of feeds. One case - each feed has own channel, server push message per channel. But if i can read 50 feed, at client we have 50 subscribers. In other case, server fetch actual user subscriptions and publish one message for all users, who have subscription.

**Comment by FZambia** [comment]
Just thought how to implement this - there are several barriers. Every channel has its own options defined over namespaces. And every message sent has unique id and channel name. So at server side we have to anyway iterate over all channels and create message for each of them, apply namespace rules and finally publish into the wire. Also what if there will be an error publishing into one of those channels - how can Centrifugo return a response for this?

When using current implementation to publish into multiple channels there are no problems described above:
```
command = [
    {"method": "publish", "params": {"channel": "channelA", "data": {"input": "xxx"}},
    {"method": "publish", "params": {"channel": "channelB", "data": {"input": "xxx"}},
    {"method": "publish", "params": {"channel": "channelC", "data": {"input": "xxx"}},
]
```
And Centrifugo just returns 3 unique response objects for each publish command in request.

So the only real benefit here is traffic and a better looking publish code, due to the logic for each channel performance gain will most probably minimal.

**Comment by aleksraiden** [comment]
Maybe only for Redis lpush publish interface (without any responce)?   If data payload has some size, and number of channels  - we have very high overhead to data transfer (and we need more logic at publish script too) for publish. 

Second idea - redis has binary-safe string, gzip/deflate data for publish?

**Comment by FZambia** [comment]
Yes, it seems reasonable for use this with Redis API only. And maybe it's better to make this as new command and call method as something like `publish_many` (or just `broadcast`) to prevent breaking semantics of current `publish` command.

**Comment by aleksraiden** [comment]
Yeah, good idea! Broadcast command sounds good!

**Comment by FZambia** [comment]
done in efc52ad6eb65aab8565ab5bb125c181df23e9ec5, it can be used over HTTP API too.
```python
from cent.core import Client
client = Client("http://localhost:8000", "secret")
client.add("broadcast", {"channels": ["public:test", "public:chat"], "data": {"input": "hello"}})
client.send()
```

It will publish data into channels until first error occurres. This error then set as response error and publishing stops. In case of using Redis API queue error will be logged. There is no currently mechanism to determine channels to which publishing was successful in case of error. Maybe I will add this as response body for `publish` and `broadcast` commands later.

**Comment by FZambia** [comment]
This will be released this in v1.2.0 and I'll also update docs
